external-crates/move/crates/move-compiler/tests/move_2024/parser/labeled_block_given_to_break_return_invalid.exp

```diff
@@ -1,3 +1,9 @@
+error[E03015]: unbound label
+  ┌─ tests/move_2024/parser/labeled_block_given_to_break_return_invalid.move:4:15
+  │
+4 │         break 'a: { 1 };
+  │               ^^ Invalid break. Unbound label 'a
```

Comment:
I don't think it's easy do to as the labeled break statement parses (and is returned) correctly - the error happens afterwards, and in general case it would be tough to tell if we should throw out the whole (correctly parsed) node or not.